apps/api/v2/src/app.module.ts

```diff
@@ -50,8 +50,6 @@ import { AppController } from "./app.controller";
     EndpointsModule,
     AuthModule,
     JwtModule,
-    //register prefix for all routes in EndpointsModule
-    RouterModule.register([{ path: "/v2", module: EndpointsModule }]),
```

Comment:
removing this since it doesn't work with custom versioning

same thing with setGlobalVersion, it's messing up with our middlewares

Reply:
@supalarry